tests/main/snapd-homedirs-vendored/task.yaml

```diff
@@ -28,6 +33,11 @@ prepare: |
     rm -rf "${unpackdir}"
 
 restore: |
+    if tests.info is-snapd-pkg-repo; then
```

Comment:
Perhaps something like:
```
test -f .skip && exit 0
```

Otherwise we may have a long piece to copy paste and keep in sync.

Reply:
updated